ledger/heavy/executor/jetkeeper.go

```diff
@@ -427,41 +428,40 @@ func (jk *dbJetKeeper) checkPulseConsistency(ctx context.Context, pulse insolar.
 	return true
 }
 
-func (jk *dbJetKeeper) all(pulse insolar.PulseNumber) []jetInfo {
+func (jk *DBJetKeeper) all(pulse insolar.PulseNumber) []jetInfo {
 	jets, err := jk.get(pulse)
 	if err != nil {
 		jets = []jetInfo{}
 	}
 	return jets
 }
 
-const (
-	jetKeeperKeyPrefix = 0x01
-	syncPulseKeyPrefix = 0x02
-)
-
-type jetKeeperKey insolar.PulseNumber
+type JetKeeperKey insolar.PulseNumber
 
-func (k jetKeeperKey) Scope() store.Scope {
+func (k JetKeeperKey) Scope() store.Scope {
 	return store.ScopeJetKeeper
 }
 
-func (k jetKeeperKey) ID() []byte {
-	return append([]byte{jetKeeperKeyPrefix}, insolar.PulseNumber(k).Bytes()...)
+func (k JetKeeperKey) ID() []byte {
+	return insolar.PulseNumber(k).Bytes()
+}
+
+func NewJetKeeperKey(raw []byte) JetKeeperKey {
+	return JetKeeperKey(binary.BigEndian.Uint32(raw))
```

Comment:
`panic: runtime error: index out of range` if len(raw) < 4

Reply:
continue, please!